openshift/template.yaml

```diff
@@ -132,3 +134,8 @@ parameters:
   required: true
   name: GUNICORN_CMD_ARGS
   value: "-b :5000 --workers=4"
+
+- description: "Subset of services to run. Use comma separated list of service ids (eg. 42,1337)"
+  value:
+  name: SERVICES_LIST
```

Comment:
And just to be 200% sure, you're comfortable having those service IDs in https://github.com/openshiftio/saas-analytics/blob/master/bay-services/3scale-admin-gateway.yaml, where anybody can see them - so they are not secrets. Just asking, please confirm @samuzzal-choudhury @invincibleJai, thanks 😉

Reply:
@msrb we should be good with that we already have route and token as secrets 
CC @samuzzal-choudhury

Reply:
@msrb this env i have kept optional , looking into ways so that we can avoid it